frontend/src/lib/OutputArtifactLoader.ts

```diff
@@ -311,6 +311,41 @@ export class OutputArtifactLoader {
         return buildArtifactViewer(script);
       }),
     );
+    const EvaluatorArtifactUris = filterArtifactUrisByType(
+      'ModelEvaluation',
```

Comment:
please do a real test via build a container image and edit yaml to put it in a real cluster to see whether it works.
harded coded codes can't be checked without a real test.

Reply:
Tested on my deployment

Reply:
Done